api/napi/__test__/js_value_conversion_global.spec.ts

```diff
@@ -0,0 +1,409 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+import test from 'ava';
+const path = require('node:path');
+var Jimp = require("jimp");
+
+import { ComponentCompiler, Brush, Model, Color, ImageData } from '../index'
+
+test('get/set string properties', (t) => {
```

Comment:
I think duplicating _all_ tests from `js_value_conversion` is overkill. That'll be a lot of extra work to maintain for no additional test coverage benefits. `js_value_conversion.spec.ts` is - as the name says - about testing conversion between JavaScript and Slint properties.

The tests here should not be about value conversion (that's already covered), it should be about property and callback access to globals. So I suggest to create one test that accesses say a string or number property of a global, tests that exceptions are thrown when getting the name of the global wrong, checks the error message when setting a non-existent property on an existing global, etc.. And then do the same with a second test about callbacks.

Reply:
done.